.github/workflows/learn-github-actions.yml

```diff
@@ -0,0 +1,13 @@
+name: learn-github-actions
+run-name: ${{ github.actor }} is learning GitHub Actions
+on: [push]
+jobs:
+  check-bats-version:
+    runs-on: ubuntu-latest
```

Comment:
- property of the job.
- Configures the job to run on the latest version of an Ubuntu Linux runner.
  -  job will execute on a fresh virtual machine hosted by GitHub
  -  For syntax examples using other runners, see "[Workflow syntax for GitHub Actions](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on)"